rasa/core/test.py

```diff
@@ -620,12 +621,14 @@ def plot_story_evaluation(
     utils.dump_obj_as_json_to_file(os.path.join(output, "results.json"), num_correct)
 
 
-def plot_curve(output: Text, no_stories: List[int]) -> None:
-    """Plot the results from run_comparison_evaluation.
+def plot_curve(output: Text, no_examples: List[int], mode: Text = "core") -> None:
```

Comment:
mhm, the mode param feels like a code smell to me. Can we have two functions or is that very verbose?